plugins/codemirror/dialogs/codemirrordialog.js

```diff
@@ -43,9 +43,17 @@ CKEDITOR.dialog.add('codemirrordialog', function (editor) {
 			preview.setSize('width', defaultWidth);
 
 			var iframe = this._createContentIframe(preview);
+
 			var iframeBody = iframe.$.contentDocument.body;
 
-			iframeBody.innerHTML = codeMirrorEditor.getValue();
+			if (CKEDITOR.env.gecko) {
```

Comment:
Hey! The fact that we need to add this kind of condition here makes me think if this is the actual problem or if it's lying a little deeper.

The `load` event should be fired in the same way in both browsers, so maybe we're adding the listener too late?

Reply:
This condition is added because `load` event in `iframe` is handled differently in Chrome/Safari and Firefox.
The issue is when appending iframe to document.
Due to multiple use cases I found while researching this, we cannot fire this event the same way to make it work in all browsers.

Reply:
Did you try adding the listener to the iframe's `onload` event before appending it to the document?

Reply:
@carloslancha is right we can handle this the same way both in Chrome or Firefox (and don't forget to check IE11 please), I checked before comenting on #133